### ISSUE TITLE
[Other General Issues]训练自己训练集，推理时图片完全检测不出来

### ISSUE THREAD

**Issue by kingkingofall** [PaddlePaddle/PaddleDetection#5341]
数据集是kaggle上的70张的 手枪和剑 目标检测数据集

训练代码：
!python -m paddle.distributed.launch tools/train.py -c configs/yolov3/yolov3_mobilenet_v1_ssld_270e_voc.yml \
                            -o weights=output/yolov3_mobilenet_v1_ssld_270e_voc/model_final.pdparams \
                            --eval
训练结果：
[03/09 14:09:36] ppdet.engine INFO: Epoch: [269] [0/6] learning_rate: 0.000010 loss_xy: 2.230445 loss_wh: 0.576769 loss_obj: 3.632374 loss_cls: 0.339784 loss: 6.829461 eta: 0:00:08 batch_cost: 1.3080 data_cost: 0.5845 ips: 6.1163 images/s
[03/09 14:09:42] ppdet.utils.checkpoint INFO: Save checkpoint: output/yolov3_mobilenet_v1_ssld_270e_voc
[03/09 14:09:42] ppdet.engine INFO: Eval iter: 0
[03/09 14:09:44] ppdet.metrics.metrics INFO: Accumulating evaluatation results...
[03/09 14:09:44] ppdet.metrics.metrics INFO: mAP(0.50, 11point) = 71.03%
[03/09 14:09:44] ppdet.engine INFO: Total sample number: 24, averge FPS: 9.418394009625983
[03/09 14:09:44] ppdet.engine INFO: Best test bbox ap is 0.710.
[03/09 14:09:46] ppdet.utils.checkpoint INFO: Save checkpoint: output/yolov3_mobilenet_v1_ssld_270e_voc
INFO 2022-03-09 14:09:50,245 launch.py:311] Local processes completed.

推理代码：
!python tools/infer.py -c configs/yolov3/yolov3_mobilenet_v1_ssld_270e_voc.yml \
                    --infer_dir=dataset/my/JPEGImages \
                    --output_dir=infer_output/ \
                    --draw_threshold=0.7 \
                    -o weights=output/yolov3_mobilenet_v1_ssld_270e_voc/best_model

但是推理图片里完全没有框出来

**Comment by yghstill** [comment]
draw_threshold设置小点再试下

**Comment by kingkingofall** [comment]
但我试过设置0.4  0.5，测试图片也还是完全没有检测框

**Comment by kingkingofall** [comment]
很奇怪，draw_threshold设置为了0，然后只有剑的目标被检测出来了，枪的目标完全没有检测到
